src/components/Chips/ChipsListingSection.jsx

```diff
@@ -1,15 +1,23 @@
 import "./chips.css";
-import { useCategory } from "../../contexts/";
+import { useCategory, useFilter } from "../../contexts/";
 import { Chip } from "../";
 
 const ChipsListingSection = () => {
 
     const { msCategories } = useCategory();
+    const { stateOfVideosFilter, dispatchOfVideosFilter } = useFilter();
 
     return (
         <div className="ms_chips-listing-section">
             <div className="ms_chips-container">
-                <div className="ms_chip b-rad1 ms_cp">
+                <div className="ms_chip b-rad1 ms_cp"
+                    onClick={() => dispatchOfVideosFilter({ type: "CLEAR" })}
+                    style={{
+                        backgroundColor: stateOfVideosFilter.filteredCategories.length === 0 && stateOfVideosFilter.filteredArtists.length === 0 ?
```

Comment:
Use the `&&` condition just once and return both bgColor & color in an object

Reply:
okk will do it!!